run_agent.sh

```diff
@@ -60,6 +60,14 @@ if [ "$AGENT_ENVIRONMENT" = "preview" ] && [ -n "$PR_NUMBER" ]; then
         cd "$AGENT_DIR"
         cp ../.env .
         poetry install
+        
+        if [ -d "browser" ]; then
+            echo "Starting browser service..."
+            cd browser
+            screen -dmS browser-service bash -c 'npm run start 2> browser-error.log'
+            cd ..
```

Comment:
because the package.json is in the root, we should just run `npm run browser:start` from the root